worlds/test/issue_2928_wide_angle_camera_texture_format.world

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" ?>
+<sdf version='1.7'>
```

Comment:
will this work in gazebo9? it's using sdformat6 which has sdf spec up to 1.6

Reply:
good point. I think we can downgrade that version. I'll test locally

Reply:
the test still fails even after downgrading this to `1.6` since the libsdformat6 parser ignores unrecognized elements like `<ignition:env_texture_format>` (some history in https://github.com/osrf/sdformat/issues/327#issuecomment-668325613)

so we will need a different solution on `gazebo9`

Reply:
I'm going to redact #2960 from this PR because it won't work with libsdformat6

Reply:
redacted